src/java.base/share/man/java.1

```diff
@@ -4418,8 +4418,13 @@ selected.
 \f[I]output\-options\f[R] is
 .RS
 .PP
-\f[CB]filecount=\f[R]\f[I]file\-count\f[R] \f[CB]filesize=\f[R]\f[I]file size
-with optional K, M or G suffix\f[R]
+\f[CB]filecount=\f[R]\f[I]file\-count\f[R] \f[CB]filesize=\f[R]\f[I]<file size
+with optional K, M or G suffix>\f[R] \f[CB]foldmultilines=\f[R]\f[I]<true|false>\f[R]
+.RE
+.PP
+\f[I]foldmultilines\f[R] enables to replace newline characters within
+a multiline log event with the character sequence '\\' and 'n'.
+Note that it works on file output only.
```

Comment:
How about:

> When foldmultilines is true, a log event that consists of multiple lines will be folded into a single line by escaping the newline (\\n) and backslash (\\) characters in the output. This option is available only for file outputs.

Reply:
This needs to be expanded similar to the help info. But you can elaborate on the character encoding limitation as per the CSR request.